docs/book/src/quick-start.md

```diff
@@ -11,7 +11,7 @@ This Quick Start guide will cover:
 
 - [go](https://golang.org/dl/) version v1.15+ (kubebuilder v3.0 < v3.1).
 - [go](https://golang.org/dl/) version v1.16+ (kubebuilder v3.1 < v3.3).
-- [go](https://golang.org/dl/) version v1.17+ (kubebuilder v3.3+).
+- [go](https://golang.org/dl/) version v1.17+ < v1.18 (kubebuilder v3.3+).
```

Comment:
Just a nit could you please squash the commits then we are OK here to get this one merged. 
@twilight0620 Thank you for your contribution 🥇

Reply:
done